setup.py

```diff
@@ -13,8 +13,14 @@
 
 """
 
+import re
+
 # Always prefer setuptools over distutils
 from setuptools import setup
 
 
-setup()
+VERSION = re.search(
+    r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]', open("src/mdacli/__init__.py").read()
```

Comment:
Since we use `bump2version`, this is not needed. We can directly inject the version in `pyproject.tolm` or anywhere needed.